src/ext/mlpm.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "demo-cat",
+  "version": "0.0.0",
+  "description": "Sample app generated with slush-marklogic-node",
+  "private": true,
+  "dependencies": {
+    "ml-constraints": "1.0.*"
+  }
+}
```

Comment:
i would prefer that this file is located at root then instructed to drop things within src/ext/ but there doesn't seem to be any option to do so at the moment.